### PR TITLE
Update mbclean.c

### PR DESCRIPTION
**Pull request by dwcaress**

### CODE REVIEW

File: src/utilities/mbclean.c [hunk 1]
```diff
@@ -81,1118 +81,1073 @@
...
-  int ping;
-  int beam;
-  double bath;
+	int flag;
+	int ping;
+	int beam;
+	double bath;
 };
 
-static const char program_name[] = "mbclean";
```
Comment (schwehr) [inline src/utilities/mbclean.c]: @dwcaress Why did you remove these changes?  Large string constants work better as static const and being in the global scope.

Comment (dwcaress) [reply src/utilities/mbclean.c]: I'll put that back. You and I both made changes to mbclean.c, and I had difficulty merging the changes.

Comment (schwehr) [reply src/utilities/mbclean.c]: ok.  no worries.  If you don't already know it, check out [meld](https://meldmerge.org).  It makes merging files easier some of the time
